server/chat.ts

```diff
@@ -1243,7 +1243,8 @@ export class CommandContext extends MessageContext {
 			const stack = [];
 			for (const tag of tags) {
 				const isClosingTag = tag.charAt(1) === '/';
-				const tagContent = tag.slice(isClosingTag ? 2 : 1).replace(/\s+/, ' ').trim();
+				const isAutoclose = tag.charAt(tag.length - 1) === '/';
+				const tagContent = tag.slice(isClosingTag ? 2 : 1, isAutoclose ? tag.length - 1 : undefined).replace(/\s+/, ' ').trim();
```

Comment:
```suggestion
				const contentEndLoc = tag.charAt(tag.length - 1) === '/' ? -1 : undefined;
				const tagContent = tag.slice(isClosingTag ? 2 : 1, contentEndLoc).replace(/\s+/, ' ').trim();
```

I think this is slightly better for readability.